code/modules/jobs/job_types/agent.dm

```diff
@@ -74,15 +74,17 @@
 		to_chat(M, "<b>You have not been assigned to any department.</b>")
 
 	var/set_attribute = normal_attribute_level
-	if(world.time >= 75 MINUTES) // Full facility expected
+	var/interval = SSabnormality_queue.spawned_abnos
+	var/max = GLOB.abnormality_room_spawners.len
```

Comment:
You do realize that the spawners delete and therefore get removed from the list when abnormalities spawn, right?

Reply:
I didn't but that's fucking amazing to know, I can really easily fix that by referencing the abno queue's "Room_Start" variable which is only set at round start.